tests/vcs/git/noconflict.py

```diff
@@ -0,0 +1,7 @@
+# A file with merge conflict markers
+print 'Line 1 does something'
+print 'No issues till here'
```

Comment:
Actually, this is a python 2 script used for testing. The bear is run on this file as
`coala --bear=GitConflictBear --files=noconflict.py`